<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="runParse">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="selectInput">
       <property name="text">
        <string>Pcap Dosyası Seçiniz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectOutput">
       <property name="text">
        <string>Hedef Klasör Seçiniz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="packetNum">
     <property name="text">
      <string>Paket Sayısı</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fileNum">
     <property name="text">
      <string>Dosya Sayısı</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="infoPacket">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>By Packet Number</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>By File Size</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
